### PR TITLE
Complete javascript primitives and variables exercise. - Dennis Nguyen

### PR DESCRIPTION
**Pull request by dennis-pnguyen**
javascript-primitives-andvariables

### CODE REVIEW

File: javascript-primitives-and-variables/main.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+const fullName = "Dennis Nguyen";
+let isCool = true;
+let totalPets = 4;
```
Comment (shawnkost) [inline javascript-primitives-and-variables/main.js]: These should be using `const` instead of `let`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Nicely done Dennis! Just one change requested here!

File: javascript-primitives-and-variables/main.js (hunk 1 above)
Comment (dennis-pnguyen) [reply javascript-primitives-and-variables/main.js]: Screenshot 2023-07-10 134129

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**